server.js

```diff
@@ -6217,6 +6217,45 @@ cache(function(data, match, sendBadge, request) {
 }));
 
 
+// Buildkite integration.
+camp.route(/^\/buildkite\/([^/]+)\/([^/]+)\.(svg|png|gif|jpg|json)$/,
+cache(function(data, match, sendBadge, request) {
+  var identifier = match[1];  // eg, 3826789cf8890b426057e6fe1c4e683bdf04fa24d498885489
+  var branch = match[2];  // eg, master
+  var format = match[3];
+
+  var url = 'https://badge.buildkite.com/' + identifier + '.json?branch=' + branch;
+  var badgeData = getBadgeData('buildkite', data);
+
+  request(url, function(err, res, buffer) {
+    if (err != null) {
+      badgeData.text[1] = 'inaccessible';
+      sendBadge(format, badgeData);
+      return;
+    }
+
+    try {
+      var data = JSON.parse(buffer);
+      var status = data.status;
+      if (status === 'passing') {
+        badgeData.text[1] = 'Passing';
```

Comment:
For the sake of consistency could you change these to be lowercase strings.
eg: travis-ci badge:
![travis-build](https://img.shields.io/travis/rust-lang/rust.svg?maxAge=2592000)